exercises/concept/international-calling-connoisseur/.docs/introduction.md

```diff
@@ -56,7 +56,7 @@ The [size][map-size-javadoc] method returns the number of entries.
 fruitPrices.size();  // Returns 2
 ```
 
-You can use the [keys] or [values] methods to obtain the keys or the values in a Map as a Set or collection respectively.studentScores
+You can use the [keys] or [values] methods to obtain the keys or the values in a Map as a Set or collection respectively.
 
 ```java
 fruitPrices.keys();    // Returns "apple" and "pear" in a set
```

Comment:
As mentioned in the other comment, should be `keySet`

```suggestion
fruitPrices.keySet();    // Returns "apple" and "pear" in a set
```

Could you also please check the following files for the same problem:
- [concepts/maps/introduction.md](https://github.com/exercism/java/blob/main/concepts/maps/introduction.md)
- [concepts/maps/about.md](https://github.com/exercism/java/blob/main/concepts/maps/about.md)